src/main/java/edu/kis/powp/command/Factories.java

```diff
@@ -0,0 +1,26 @@
+package edu.kis.powp.command;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Factories {
+    public static List<DriverCommand> drawRectangle(int x, int y, int a, int b) {
```

Comment:
Wrong name. The method doesn't draw.
It would be simpler if the factories would return DriverCommands